src/components/ProfilePage/components/AddingProofsForm/AddingProofsForm.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useContext } from "react";
+import { Button } from "../../../../shared/components";
+import { Input } from "../../../../shared/components";
+import { Textarea } from "../../../../shared/components";
+import { TalentsService } from "../../../../services/api-services";
+import { UserContext } from "../../../../context/UserContext";
+import s from "./AddingProofsForm.module.scss";
+export function AddingProofsForm({ active, setActive }) {
+    //const [id, setId]=useState(0)
+    const { user, token } = useContext(UserContext);
+    const [link, setLink] = useState("");
+    const [text, setText] = useState("");
+    const [talentsProofs, setTalentsProofs] = useState([]);
+    function handle(e) {
+        e.preventDefault();
+        const proof = { link, text };
+        TalentsService.addProof(proof, user.id, token)
+            .then(() => {
+                if (text !== "") {
+                    const allProofs = [...talentsProofs, proof];
```

Comment:
You can use previous state instead of create constant
```suggestion
                    setTalentsProofs((prev)=>[...prev, proof]);
```